docker-compose.yml

```diff
@@ -156,15 +156,12 @@ services:
       - AZURE_PORTAL_STORAGE_ACCESS_KEY
       - AZURE_PORTAL_FILESHARE_NAME
       - AZURE_PORTAL_EXPORT_FOLDER
-      - DTFS_PORTAL_SCHEDULER
       - GOV_NOTIFY_API_KEY
       - GOV_NOTIFY_EMAIL_RECIPIENT
       - DTFS_PORTAL_SCHEDULER=true
-      - FETCH_WORKFLOW_TYPE_B_SCHEDULE=*/1 * * * * *
```

Comment:
Can we add to the readme about this or add a comment to the referenced jira ticket? https://github.com/UK-Export-Finance/dtfs2#workflowtypeb-integration

Just incase we have to re-enable workflow

Reply:
good shout! - I'll do that